### DISCUSSION TITLE
Laravel 11 paginator (bootstrap) layout not working

### DISCUSSION THREAD

**Discussion by xciser77** [laravel/framework#50682] [General]
I am 'upgrading' my application from 10 to 11 and are copying code from my old project to the new one.
In my Laravel 10 application i have set the paginator to use the bootstrap 5 template in the AppServiceProvider.
When I do the same in Laravel 11, the new view is not parsed and the paginator is still using the tailwind version

I tried both in the Boot section of the AppServiceProvider, but both are ignored.
Paginator::useBootstrapFive();
Paginator::defaultView('vendor.pagination.bootstrap-5');

**Comment by xciser77** [comment]
No issue, i forgot I was using livewire, and need to set it different

**Reply by ajithjojo** [reply]
i am also using livewire how to set that ?

**Reply by eduardr10** [reply]
From livewire documentation
![image](https://github.com/user-attachments/assets/423faf8b-87d3-4268-8c1b-046ada559362)
